### DISCUSSION TITLE
Hide configuration title

### DISCUSSION THREAD

**Discussion by Lockev** [perspective-dev/perspective#2266] [Q&A]
Hello Perspective community,

I'm currently trying to build a Perspective Vue3 wrapper. My goal here is to create a prop that toggles the possibility for the user to open the configuration panel in order to create static client side dashboards. 

<img width="249" alt="Capture d’écran 2023-06-22 à 08 22 59" src="https://github.com/finos/perspective/assets/54106288/5baeccd3-6e32-4b9a-8928-124940477fd5">

However, I cannot find any intel on what should I toggle in my layout/settings to hide this 'configure' button. 
(Exemple of my current reactive variable that I currently have for test purposes) 
<img width="414" alt="Capture d’écran 2023-06-22 à 08 27 02" src="https://github.com/finos/perspective/assets/54106288/7bd637b4-7ec6-4ebb-87b8-c115e92fc657">

On top of that, it seems that the javascript running behind this button is super resilient and doesn't allow me to do any kind of quick patchwork :
- Remove the button from the DOM using javascript
- Hide it in css with ':host #settings_button { display: none !important } when any other css manipulation works on the table (both pure CSS and Javascript DOM edition)

Is there any currently available option that could allow me to code this in a way ? Many thanks 🙏

**Comment by texodus** [comment]
This is as-intended - the component's behavior and interaction model aren't meant to be customized. You can still trigger the config menu externally through the persistence API or it's dedicated method (though they are awkwardly-named ATM):

```js
await viewer.restore({settings: true});

// or

await viewer.toggleConfig();
```
